AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -143,20 +143,30 @@ internal class CardViewController: FormViewController {
             items.numberItem.showLogos(for: brands.map(\.type))
         }
 
-        let isHidden: Bool
+        let kcpItemsHidden: Bool
```

Comment:
`kcpItemsHidden` -> `kcpItemIsHidden`  :-)

Reply:
:D If we gotta add 'is', gotta add it to beginning right? 
Since they are local vars, figured it shouldn't matter either one.

Reply:
i thought you forgot the `I`, but its actually `kcp Items Hidden` where Items is plural :-D